src/vocab.rs

```diff
@@ -75,4 +82,13 @@ impl PySequenceProtocol for PyVocab {
             Ok(words[idx as usize].clone())
         }
     }
+
+    fn __contains__(&self, word: String) -> PyResult<bool> {
```

Comment:
Seems to be an unrelated change of another PR?

Reply:
Forgot to rebase this on master after merging the `contains` PR, will do that now.